loadtest/control/simplecontroller/controller.go

```diff
@@ -49,19 +49,19 @@ func (c *SimpleController) Run() {
 			waitAfter: 1000,
 		},
 		{
-			run:       c.createPost,
+			run:       c.reload,
```

Comment:
Since we already handle connecting and disconnecting in login/logout I think we can add a parameter to `reload()` specifying whether we want a complete reload or a first load with the only difference that in the former case we would also disconnect and connect.  
Then we can probably wrap it into an anonymous function here so we keep the action *parameterless*.